proto/rpc/v1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/transaction.proto";
+import "rpc/v1/meta.proto";
+
+message TxRequest {
+    // the hash of the transaction
+    bytes hash = 1;
+
+    // if true, return data in binary format
+    // TODO: does this make sense for protobuf?
+    bool binary = 2;
+}
+
+message TxResponse {
+
+    // The actual transaction
+    oneof sttx {
+        Transaction tx = 1;
+        bytes tx_bytes = 2;
+    };
+    // Sequence number of ledger that contains this transaction
+    uint64 ledger_index = 3;
+
+    // Hash of the transaction
+    bytes hash = 4;
+
+    // whether the ledger has been validated
+    bool validated = 5;
+
+    // metadata about the transaction
+    oneof stmeta {
+        Meta meta = 6;
+        bytes meta_bytes = 7;
```

Comment:
@seelabs would you also omit the type suffix here? I could change it to `meta_binary` , but that's still adding the type as a suffix, even if its not the exact name of the protobuf type.

Reply:
@cjcobb23 Maybe  `raw_meta`? I'm OK with `_binary` too. Name are hard, maybe someone else has a good suggestion.

Reply:
changed to `meta_binary` and `transaction_binary`